docs/.authors.yml

```
@@ -1,9 +1,45 @@
 authors:
-  dalonsoa:
-    name: Diego Alonso Álvarez
-    description: Head of RSEs
-    avatar: https://github.com/dalonsoa.png
+  AdrianDAlessandro:
+    name: Adrian D'Alessandro
+    description: Senior Research Software Engineer
+    avatar: https://github.com/AdrianDAlessandro.png
+  alexdewar:
+    name: Alexander Dewar
+    description: Senior Research Software Engineer
+    avatar: https://github.com/alexdewar.png
+  HarmonicReflux:
+    name: Benjamin J Scharpf
+    description: HPC and RSE Experience Programme
+    avatar: https://github.com/HarmonicReflux.png
   cc-a:
     name: Chris Cave-Ayland
     description: RSE Technical Lead
     avatar: https://github.com/cc-a.png
+  dc2917:
+    name: Daniel Cummins
+    description: Research Software Engineer
+    avatar: https://github.com/dc2917.png
+  dalonsoa:
+    name: Diego Alonso Álvarez
+    description: Head of RSEs
+    avatar: https://github.com/dalonsoa.png
+  jamesturner246:
+    name: James Turner
+    description: Senior Research Software Engineer
+    avatar: https://github.com/jamesturner246.png
+  TinyMarsh:
+    name: Ryan Smith
+    description: Senior Research Software Engineer
+    avatar: https://github.com/TinyMarsh.png
+  Sahil590:
+    name: Sahil Raja
+    description: Junior Software Developer
+    avatar: https://github.com/Sahil590.png
+  SaranjeetKaur:
+    name: Saranjeet Kaur Bhogal
+    description: Research Software Engineer
+    avatar: https://github.com/SaranjeetKaur.png
+  tsmbland:
+    name: Tom Bland
+    description: Research Software Engineer - HPC and RSE Experience Programme
```

We are about to change his job title.
```suggestion
    description: Research Software Engineer
```